src/backend/expungeservice/models/record_object.py

```diff
@@ -0,0 +1,21 @@
+from expungeservice.models.case import Case
+
+class Record:
```

Comment:
There is nothing wrong with the Record class. You did an excellent job! I can see it will work as intended. One thing that we can do, that I just learned about, is to use the `@property` decorator on the `charges` method. This will make the method act like an attribute so that we can ask `Record.charges` instead of `Record.charges()` -> Obviously not that big of an issue, but if you'd like to see how it can be used here is a good site: https://www.programiz.com/python-programming/property

Reply:
One thing I would like to do is change all the current code in master to use the `@property` tag. But it's not a priority at this time. However I would like to see new code use... That is why I bring it up.